src/javascript/app/Modules/Reports/Helpers/shortcode.js

```diff
@@ -1,27 +1,28 @@
 const Shortcode = (() => {
-    let info_from_shortcode = {};
+    const info_from_shortcode = {
```

Comment:
I'm not sure it's a good idea to have this global. it's something that gets overwritten all the time since it's specific to each time the function gets called not to the page etc. so better have it inside the function.